pkg/kube/snapshot/snapshot_test.go

```diff
@@ -1114,10 +1118,10 @@ func (s *SnapshotTestSuite) TestLabels(c *C) {
 		{
 			dynCli: dynfake.NewSimpleDynamicClient(scheme),
 			createLabels: map[string]string{
-				"label": "1",
+				"label": "1/2/3",
 			},
 			listLabel: map[string]string{
-				"label": "1",
+				"label": "1/2/3",
```

Comment:
For my clarification - You added the `/`  because the Sanitize API converts them to `_` correct ?

Reply:
yep